commercetools-models/src/main/java/io/sphere/sdk/customers/commands/updateactions/ChangeAddress.java

```diff
@@ -18,22 +18,24 @@
 public final class ChangeAddress extends UpdateActionImpl<Customer> {
     private final Address address;
     private final String addressId;
+    private final String addressKey;
 
-    private ChangeAddress(final String addressId, final Address address) {
+    private ChangeAddress(final String addressId, final Address address, String addressKey) {
         super("changeAddress");
         this.address = address;
         this.addressId = addressId;
+        this.addressKey = addressKey;
     }
 
-    public static ChangeAddress of(final String addressId, final Address address) {
-        return new ChangeAddress(addressId, address);
+    public static ChangeAddress of(final String addressId, final String addressKey, final Address address) {
```

Comment:
Please never change existing factory methods. Introduce a new one with the new parameter. Else this is a breaking change for existing applications